### DISCUSSION TITLE
Sort a table

### DISCUSSION THREAD

**Discussion by rafikApp** [Budibase/budibase#8168] [Help]
Hi everyone.
when I click on a column of the table to sort it, the table empties.
Any solutions please
Ps : I have already added a filter condition

**Comment by andz-bb** [comment]
hey @rafikApp, could you send your app export to support@budibase.com please? I can take a closer look today

**Reply by rafikApp** [reply]
Hi @andz-bb 
it's ok i sent it

**Reply by andz-bb** [reply]
hey @rafikApp, I imported your app but I don't think I can replicate what you are seeing. When I sort on each of the stock table columns they work correctly and all of the results remain in the table. Could you check in your developer tools network tab or console (F12 on chrome) to see if there are any errors for you when you try to sort? Thanks
